src/Midgard/CreatePHP/Metadata/AbstractRdfDriver.php

```diff
@@ -72,19 +72,6 @@ protected function buildInformation($child, $identifier, $field, &$add_default_v
     protected abstract function getAttributes($element);
 
     /**
-     * {@inheritDoc}
-     *
-     * The default implementation uses get_class on objects
-     */
-    public function objectToName($object, RdfMapperInterface $mapper)
```

Comment:
i agree with moving this to the mapper.

but i suggest we add an `@deprected` to this method and simply call $mapper->objectToName , rather than doing the BC break now.